framework/examples/myproject/go.mod

```diff
@@ -3,9 +3,8 @@ module github.com/smartcontractkit/chainlink-testing-framework/framework/example
 go 1.22.7
 
 replace (
-	github.com/smartcontractkit/chainlink-testing-framework/framework => ../..
-	github.com/smartcontractkit/chainlink-testing-framework/framework/cmd => ../../cmd/
-	github.com/smartcontractkit/chainlink-testing-framework/framework/wasp => ../../wasp/
+	github.com/smartcontractkit/chainlink-testing-framework/framework => ../../
+	github.com/smartcontractkit/chainlink-testing-framework/wasp => ../../../wasp
```

Comment:
These were invalid and tripping up the graph generation, so I fixed them and tidies the whole repo.

Reply:
That's needed too
```
github.com/smartcontractkit/chainlink-testing-framework/framework/cmd => ../../cmd/
```

Reply:
If it's blocking your tool I can remove them and stick with tags @jmank88 , these are e2e examples and it's okay to update them if you want to remove replaces at all

Reply:
What do you mean? There is no `cmd/go.mod` :shrug:

Reply:
Oh, okay 😄